src/Directory.Build.props

```diff
@@ -1,3 +1,9 @@
 <Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'OpenTelemetry.sln'))\build\Common.prod.props" />
+  <PropertyGroup>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
```

Comment:
if we simply put the readme.md file in the nuget, it won't work as there are many relative links to this readme.md file, which'll be broken.